<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addElement</class>
 <widget class="QDialog" name="addElement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>405</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Add Element</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="addElementBox">
     <item>
      <property name="text">
       <string>Phototransistor</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Voltage Source</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Schottky Diode</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Battery</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="xLabel">
       <property name="text">
        <string>X-coordinate:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="xSpinBox">
       <property name="maximum">
        <number>2000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="yLabel">
       <property name="text">
        <string>Y-coordinate:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="ySpinBox">
       <property name="maximum">
        <number>2000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="colourLabel">
       <property name="text">
        <string>Colour:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="colourBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>azure</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>blue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>brown</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>gray</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>green</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>orange</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>pink</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>red</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>violet</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>yellow</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lineSizeLabel">
       <property name="text">
        <string> Line size:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="lineSizeBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="elementRadiusLabel">
       <property name="text">
        <string>Element radius:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="elementRadiusSpinBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>25</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="elementPriceLabel">
       <property name="text">
        <string>Element price [$]&quot;:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="elementPriceSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="currentLabel">
       <property name="text">
        <string>Current:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="currentComboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Alternating Current (A.C.)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Direct Current (D.C.)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="elementPowerLabel">
       <property name="text">
        <string>ElementPower [V]:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QSpinBox" name="elementPowerPinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="elementTimeLabel">
       <property name="text">
        <string>Time to charge:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QSpinBox" name="elementTimePinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="okPushButton">
     <property name="text">
      <string>Add</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>addElementBox</tabstop>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>xSpinBox</tabstop>
  <tabstop>ySpinBox</tabstop>
  <tabstop>colourBox</tabstop>
  <tabstop>lineSizeBox</tabstop>
  <tabstop>elementRadiusSpinBox</tabstop>
  <tabstop>elementPriceSpinBox</tabstop>
  <tabstop>currentComboBox</tabstop>
  <tabstop>elementPowerPinBox</tabstop>
  <tabstop>elementTimePinBox</tabstop>
  <tabstop>okPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
